tools/pythonpkg/tests/fast/spark/test_spark_column.py

```diff
@@ -15,11 +16,19 @@ class TestSparkColumn(object):
     def test_struct_column(self, spark):
         df = spark.createDataFrame([Row(a=1, b=2, c=3, d=4)])
```

Comment:
Can you add a FIXME to this test? I think we want to fix this behavior in the future, this is exactly the type of thing that these tests against (actual) pyspark are great for finding 👍 

`FIXME: column names should be set explicitly using the Row, rather than letting duckdb assign defaults (col0, col1, etc..)`